### PR TITLE
fix(version) : add some punctuation

### PR DESCRIPTION
**Pull request by hwdef**
Signed-off-by: hwdef <hwdef97@gmail.com>

### Ⅰ. Describe what this PR did

add some punctuation in version and .gitignore

### Ⅱ. Does this pull request fix one issue?

NONE

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -35,33 +35,33 @@ import (
 
 var (
 	// version is the version of project Dragonfly
-	// populate via ldflags
+	// populate via ldflags.
```
Comment (Starnop) [inline version/version.go]: No need to add dot here.

File: version/version.go [hunk 2]
```diff
@@ -35,33 +35,33 @@ import (
...
 	version string
 
 	// revision is the current git commit revision
-	// populate via ldflags
+	// populate via ldflags.
```
Comment (Starnop) [inline version/version.go]: ditto

File: version/version.go [hunk 3]
```diff
@@ -35,33 +35,33 @@ import (
...
 	revision string
 
 	// buildDate is the build date of project Dragonfly
-	// populate via ldflags
+	// populate via ldflags.
```
Comment (Starnop) [inline version/version.go]: ditto

**Comment by hwdef** [comment]
@Starnop 
I fixed it.

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**